sections/futures/FeeInfoBox/FeeInfoBox.tsx

```diff
@@ -138,7 +138,7 @@ const FeeInfoBox: React.FC = () => {
 									i18nKey={`dashboard.stake.tabs.trading-rewards.stake-to-${
 										isRewardEligible ? 'earn' : 'start'
 									}`}
-									components={[<Emphasis />]}
+									components={[<Text.Body variant="bold" />]}
```

Comment:
Be good to stick to the convention of importing the specific component so we'd just have <Body>

Reply:
Good point. Updated.